framework/packages/abstract-core/src/core/manager.rs

```diff
@@ -167,6 +168,10 @@ pub enum ExecuteMsg {
         description: Option<String>,
         // URL linked to the account
         link: Option<String>,
+        // Optionally specify a base asset for the sub-account
+        base_asset: Option<AssetEntry>,
```

Comment:
Would be nice if we can have default to parent base_asset in case it's `None`. Don't think it's implementable now